js/modules/modules.go

```diff
@@ -95,20 +95,14 @@ type VU interface {
 	// Runtime returns the goja.Runtime for the current VU
 	Runtime() *goja.Runtime
 
-	// RegisterCallback lets a module declare that it wants to run a function on the event loop *at a later point in time*.
-	// It needs to be called from within the event loop, so not in a goroutine spawned by a module.
-	// Its result can be called with a function that will be executed *on the event loop* -
-	// possibly letting you call RegisterCallback again.
-	// Calling the result can be done at any time. The event loop will block until that happens, (if it doesn't
-	// have something else to run) so in the event of an iteration end or abort (for example due to an exception),
-	// It is the module responsibility to monitor the context and abort on it being done.
-	// This still means that the returned function here *needs* to be called to signal that the module
-	// has aborted the operation and will not do anything more, not doing so will block k6.
+	// RegisterCallback lets a JS module declare that it wants to run a function
+	// on the event loop *at a later point in time*. See the documentation for
+	// `EventLoop.RegisterCallback()` in the `k6/js/eventloop` Go module for
+	// the very important details on its usage and restrictions.
```

Comment:
```suggestion
	// RegisterCallback lets a JS module declare that it wants to run a function
	// on the event loop *at a later point in time*. See the documentation for
	// [*go.k6.io/k6/js/eventloop/EventLoop.RegisterCallback]  for
	// the very important details on its usage and restrictions.
```

Not certain how well this works but apperntly this is the [syntax for doc links](https://go.dev/doc/comment) (need to search for "doc links" as there are no anchors :( ) It seems we might drop everything before `eventloop` too :thinking: but we need to probably test this.

Reply:
Uff .. thsi won't work for this as this is a method on an interface >.>